boards/posix/native_posix/CMakeLists.txt

```diff
@@ -12,6 +12,15 @@ zephyr_library_sources(
 	cmdline.c
 	)
 
+if(CONFIG_HAS_SDL)
+	find_package(PkgConfig REQUIRED)
```

Comment:
For the cmake parts better if @SebastianBoe would take a look, but it looks a bit funny to have this code almost being a replica of the one for the driver.